### DISCUSSION TITLE
WW3 latest version

### DISCUSSION THREAD

**Discussion by fed1942** [NOAA-EMC/WW3#916] [Q&A] [answered]
What is the latest version of WW3 that is available for download?. Someone told me that there is a 7.XX version but on the NOAA website it is mentioned that version 6.07 is the latest.

Thank you in advance.

**Comment by aliabdolali** [comment, marked as answer]
The develop branch is the latest.
The public release is 6.07 which is a few years behind.

**Reply by EmilioEchevarria** [reply]
@aliabdolali I was going to ask a similar question. Is there a plan to tag and release version 7.XX in the near future? It might be useful in terms of replicating results, or errors. Many papers are already citing WW3 v7.XX, and it might not be very clear exactly which state of this Github repository they used.

**Reply by EmilioEchevarria** [reply]
I'm aware of this: https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number

But I don't know how strict that is, I couldn't find commits for all the versions listed [here](https://github.com/NOAA-EMC/WW3/wiki/About-WW3)

**Reply by aliabdolali** [reply]
@MatthewMasarik-NOAA and @JessicaMeixner-NOAA can give you an accurate answer about the release version.

**Reply by JessicaMeixner-NOAA** [reply]
Right now we're waiting for a few more developments to be finished before moving forward with another release tag.   Currently not sure of what that development time frame is as we'd actually hoped to have that done quite some time ago.

**Reply by iurnus** [reply]
Are there any news about the newer version? I have used v6.07 in my coupled model and I am looking forward for newer versions. It been a long time since the last tagged version.

**Reply by JessicaMeixner-NOAA** [reply]
There are no immediate plans for a new tagged version, however the develop branch is available for your use at anytime.

**Comment by aronroland** [comment]
Hi All, 

 

I think this is very important question and I think we need more tight release tags … as for reproducibility. 

 

Cheers

 

Aron

 

Von: Ali.Abdolali ***@***.***> 
Gesendet: Freitag, 24. März 2023 06:52
An: NOAA-EMC/WW3 ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: Re: [NOAA-EMC/WW3] WW3 latest version (Discussion #916)

 

@MatthewMasarik-NOAA <https://github.com/MatthewMasarik-NOAA>  and @JessicaMeixner-NOAA <https://github.com/JessicaMeixner-NOAA>  can give you an accurate answer about the release version.

**Reply by JessicaMeixner-NOAA** [reply]
@aronroland please see reply above.   I will bring your point of more frequent releases to the next code managers tag-up.
